README.md

```diff
@@ -23,20 +24,20 @@ under the License.
 
 # Device-Management Repository
 
-This Repo contains the code for Device-Manager and related functionality. Device-manager is module which collects the
-device data from the devices which support REDFISH and publishes onto kafka bus. User-application is another software
-which listens on kafka bus and makes the data available to the dashboard for user.
+This repository contains the code for Device-Manager and related functionality.
```

Comment:
Device Manager instead of Device-Manager

Reply:
Done.